src/Symfony/Component/Messenger/Transport/RedisExt/Connection.php

```diff
@@ -43,6 +43,9 @@ class Connection
     private $consumer;
     private $autoSetup;
     private $maxEntries;
+    private $redeliveryTime = 600000;
```

Comment:
@toooni think we should use the same value here as we used for doctrine connection doctrine uses 3600 sec so I would use the same value:

https://github.com/symfony/symfony/blob/07818f2747548a1a88e7339c6148f26537c5174f/src/Symfony/Component/Messenger/Transport/Doctrine/Connection.php#L36



And would make `redelivery_time` and `claim_interval` configureable over the dsn so you can set `?redelivery_time=3600&claimInterval